### PR TITLE
replaceAll-workaround

### PR DESCRIPTION
**Pull request by haslinghuis**
- [x] Fixes an issue on Cordova Android

Credit to @SEMICS https://stackoverflow.com/a/72049058
Tested by: @RabbitAmbulance-RA 

Co-authored-by: @nerdCopter <56646290+nerdCopter@users.noreply.github.com>

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by nerdCopter** [comment]
Should this be limited to Android only? 
maybe something like
```
    if (/android/i.test(userAgent)) {
        [code]
    }
```

**Comment by nerdCopter** [comment]
Didn't seem to break anything on Linux, but only lightly tested. Might prefer limiting to Android just in case as stated above.

**Comment by haslinghuis** [comment]
Yes tried yesterday on line 187 - adding now

**Comment by nerdCopter** [comment]
> Do you want to test this code? Here you have an automated build: [Betaflight-Configurator-Android](https://github.com/betaflight/betaflight-configurator/suites/11074033953/artifacts/563373102)

I was able to use it on an Android 9 (Jan 2020) phone. Basic PIDs/Ports/CLI tabs. Did not test others.

**Review by blckmn — APPROVED** [review]

File: src/js/main.js
Comment (chmelevskij) [inline src/js/main.js:29]: I don't think we need that android test tbh. `typeof` sort of covers all the required bases imo.

Comment (nerdCopter) [reply src/js/main.js:29]: thank you; sounds fair. if that's the case @haslinghuis can revert and it can just be merged since it was tested the other way already,

**Review by nerdCopter — APPROVED** [review]

**Pull request merged**
